pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -220,10 +232,52 @@ func sortedKeys(m map[string]string) []string {
 	return keys
 }
 
-// getUserData returns the exact same string for equivalent input,
+func (p *LaunchTemplateProvider) getBottleRocketUserData(ctx context.Context, constraints *v1alpha1.Constraints, additionalLabels map[string]string) (string, error) {
```

Comment:
Could you also support maxPods as that was functionality added in [this PR](https://github.com/aws/karpenter/pull/1032/files). 

For BR, it should be `settings.kubernetes.max-pods`

Reply:
This could be a follow-up change too :)